katalog/istio/README.md

```diff
@@ -17,169 +34,175 @@ Source: [https://istio.io/docs](https://istio.io/docs/concepts/what-is-istio/#wh
+
+```yaml
+namespace: istio-system
+
+bases:
+  - katalog/istio/init
+  - katalog/istio
+  - katalog/istio/telemetry
 ```
 
-Then, **restart the kiali pod** to take effect.
+#### Service Monitors
 
-##### Prometheus URL
+As telemetry exposes metrics in prometheus format, this package provides [service-monitor](telemetry/service-monitor) definition for prometheus operator.
 
-There is a minor configuration to be made if you don't have the default
-[`prometheus-operated`](https://github.com/sighupio/fury-kubernetes-monitoring/tree/v1.3.0/katalog/prometheus-operated)
-server deployed in the `monitoring` namespace.
+Example kustomize installation file:
 
-As Kiali needs to read metrics from a prometheus server, you have to define the prometheus server url.
+```yaml
+namespace: istio-system
 
-```bash
-$ kubectl get cm kiali -o yaml -n istio-system
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  labels:
-    app: kiali
-  name: kiali
-  namespace: istio-system
-data:
-  config.yaml: |
-    istio_namespace: istio-system
-    deployment:
-      accessible_namespaces: ['**']
-    auth:
-      strategy: login
-    server:
-      port: 20001
-      web_root: /kiali
-    external_services:
-      tracing:
-        url:
-      grafana:
-        url:
-      prometheus:
-        url: http://prometheus-k8s.monitoring:9090
+bases:
+  - katalog/istio/init
+  - katalog/istio
+  - katalog/istio/telemetry/service-monitor
 ```
 
-Change the `external_services.prometheus.url` value with the correct prometheus server url.
+##### Note
+
+This package will deploy telemetry + service monitors. If telemetry it's already deployed, replace the telemetry base
+with the `katalog/istio/telemetry/service-monitor` base.
+
+
+### Egress Gateway
+
+This package deploys on top of the [minimal installation](#minimal-installation),
+[the egress gateway component](egress-gateway/README.md).
+
+Once deployed this package, don't forget to deploy the new Istio configuration based on the components installed:
+
+[Example kustomize installation file](../../examples/istio/minimal-and-egress/kustomization.yaml):
+
+```yaml
+namespace: istio-system
+
+bases:
+  - katalog/istio/init
+  - katalog/istio/minimal
+  - katalog/istio/egress-gateway
+  - katalog/istio/sidecar-injection/configuration/minimal-and-egress
+```
+
+### Citadel
+
+[Citadel](citadel/README.md) is the Istio component in charge of provide security features *(key and certificate management)*.
+If you are looking for mTLS, this is the component you should install.
+
+As the other components, this one should be installed on top of the minimal istio deployment.
+
+Example kustomize installation file:
+
+```yaml
+namespace: istio-system
+
+bases:
+  - katalog/istio/init
+  - katalog/istio/minimal
+  - katalog/istio/citadel
+  - katalog/istio/egress-gateway
+  - katalog/istio/sidecar-injection/configuration/citadel-and-egress
+```
+
+In this example you can see istio + citadel + egress gateway installed.
```

Comment:
You are referencing to the previous kustomize piece, right? maybe refrase this to:
"With the previous example snippet you will get `istio` with `citadel` and `egress gateway` modules installed."

Reply:
Solved!